lib/src/models/movie_model.dart

```diff
@@ -0,0 +1,29 @@
+import 'result_movie_model.dart';
+
+class MovieModel {
```

Comment:
The class itself denotes a Model, since it is also in the `models` package, you can just name it Movie, what do you think ? Also, lets talk about using packages in plural or singular but it should be a practice used all around your proyect.

Reply:
Just 'movie' would be OK since what you said about the packege name refers 'Model' from iteself. Then I changed 'MovieModel' class name to just 'Movie'. About using packages names in plural or singular, what do you think it would be better to use from now on?